lib/src/navbar/navbar.component.scss

```diff
@@ -176,3 +221,10 @@ $navbar-fixed-shadow: 0px 2px 6px $shadow;
         .navbar-app { padding: 0 0 0 5%; }
     }
 }
+
+.more-popover a {
+    &.active {
+        color: $text;
```

Comment:
The active link being set to black looks strange, did you mean to do this?

![image](https://user-images.githubusercontent.com/1934453/38685085-572d6362-3e2e-11e8-9da3-8b8e397b4509.png)